Physical_Engine/Query/InternalPolyline.cs

```diff
@@ -54,6 +54,12 @@ public static List<Polyline> InternalPolyline(IPhysical physicalObject)
         [Output("polyline", "BHoM Geometry Polyline")]
         public static List<Polyline> InternalPolyline(BH.oM.Physical.Elements.ISurface physicalObject)
         {
+            if(physicalObject == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the internal polyline of a null surface.");
+                return new List<Polyline>();
```

Comment:
As above, shouldn't a query on `null` be returning a `null`? Refers to all such cases in this PR (and beyond it 😉).

Reply:
See above response :smile: